recipes/_compute_slurm_config.rb

```diff
@@ -15,6 +15,13 @@
 # OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Create directory configured as SlurmdSpoolDir
+directory '/var/spool/slurmd.spool' do
```

Comment:
Why don't use the default folder `/var/spool/slurmd` as mentioned in the [documentation](https://slurm.schedmd.com/slurm.conf.html)?

Reply:
this one I changed it to `/var/spool/slurmd`